lib/t/push_notifications/dispatch_job.ex

```diff
@@ -73,15 +81,28 @@ defmodule T.PushNotifications.DispatchJob do
     %{"match_id" => match_id, "receiver_id" => receiver_id, "slot" => slot} = args
 
     if match = alive_match(match_id) do
+      %Matches.Match{user_id_1: uid1, user_id_2: uid2} = match
+
       timeslot =
         Matches.Timeslot |> where(match_id: ^match_id, selected_slot: ^slot) |> Repo.one()
 
       if timeslot do
         Matches.schedule_timeslot_ended(match, timeslot)
 
-        receiver_id
-        |> Accounts.list_apns_devices()
-        |> schedule_apns(type, %{"match_id" => match_id})
+        picker_id =
+          if(receiver_id == uid1) do
+            uid2
+          else
+            uid1
+          end
```

Comment:
same here and everywhere below

Reply:
this push was not in the last PR at all, not good

Reply:
yeah, decide to add too